File: app-java/src/main/java/com/eress/closer/util/ApplicationUtil.java
package com.eress.closer.util;

import android.app.Application;
import android.content.res.Configuration;

public class ApplicationUtil extends Application {

    public int intro = 0;
    public int flag = 0;

    @Override
    public void onCreate() {
        super.onCreate();
    }

    @Override
    public void onConfigurationChanged(Configuration newConfig) {
        super.onConfigurationChanged(newConfig);
    }
}

File: app-kotlin/src/main/kotlin/com/eress/closer/util/ApplicationUtil.kt
package com.eress.closer.util

import android.app.Application
import android.content.res.Configuration

class ApplicationUtil : Application() {

    var intro = 0
    var flag = 0

    override fun onCreate() {
        super.onCreate()
    }

    override fun onConfigurationChanged(newConfig: Configuration) {
        super.onConfigurationChanged(newConfig)
    }
}
